data/sql_updates/create_candidates_view.sql

```diff
@@ -25,13 +25,21 @@ select
     max(dimparty.party_affiliation) as party,
     max(dimparty.party_affiliation_desc) as party_full,
     max(dimoffice.office_state) as state,
-    (select cand_nm from dimcandproperties cp where cp.cand_sk = dimcand.cand_sk order by candproperties_sk desc limit 1) as name
+    max(candprops.cand_nm) as name
 from dimcand
     left join (select distinct on (cand_sk) cand_sk, election_yr, cand_status, ici_code from dimcandstatusici order by cand_sk, election_yr desc) csi_recent using (cand_sk)
     left join dimcandstatusici csi_all using (cand_sk)
     inner join dimcandoffice co on co.cand_sk = dimcand.cand_sk and (csi_recent.election_yr is null or co.cand_election_yr = csi_recent.election_yr)  -- only joined to get to dimoffice
     inner join dimoffice using (office_sk)
     inner join dimparty using (party_sk)
+    left join (
```

Comment:
@LindsayYoung OK, we can revert that last change or think of a more efficient way to write the query without changing the results. Can you confirm that sorting on `candproperties_sk` instead of `cand_sk` gives us the most recent name for each candidate?